samcli/commands/local/lib/generated_sample_events/events/apigateway/AwsProxy.json

```diff
@@ -3,7 +3,7 @@
   "resource": "{{resource}}",
   "path": "/{{path}}",
   "httpMethod": "{{method}}",
-  "isBase64Encoded": "false",
+  "isBase64Encoded": false,
```

Comment:
This should actually be `true` since SAM CLI does convert the data to Base64

Reply:
The value should reflect the actual event. And in this case, that means this field should be a bool and `true` since the event data is base64 encoded.